src/Helpers/constants.js

```diff
@@ -111,14 +110,22 @@ export const CVSS_OPTIONS = [
     { value: 'from8to10', label: '8.0 - 10.0', from: 8, to: 10 }
 ];
 
+const substractDays = (toSubstract, currDate = new Date()) => {
+    return currDate.setDate(currDate.getDate() - toSubstract);
+};
+
+const substractYears = (toSubstract, currDate = new Date()) => {
+    return currDate.setFullYear(currDate.getFullYear() - toSubstract);
+};
+
 //Public date labels to value
 export const PUBLIC_DATE_OPTIONS = [
     { value: 'all', label: intl.formatMessage(messages.optionsAll) },
-    { value: 'last7', label: intl.formatMessage(messages.lastxdays, { days: 7 }), from: moment().subtract(7, 'days') },
-    { value: 'last30', label: intl.formatMessage(messages.lastxdays, { days: 30 }), from: moment().subtract(30, 'days') },
-    { value: 'last90', label: intl.formatMessage(messages.lastxdays, { days: 90 }), from: moment().subtract(90, 'days') },
-    { value: 'lastYear', label: intl.formatMessage(messages.lastYear), from: moment().subtract(1, 'years') },
-    { value: 'MoreThanYear', label: intl.formatMessage(messages.moreThanYear), to: moment().subtract(1, 'years') }
+    { value: 'last7', label: intl.formatMessage(messages.lastxdays), from: substractDays(7) },
```

Comment:
oups,  I guess you accidentally removed the  `intl.formatMessage(messages.lastxdays, { days: X })` which is needed to format the message 

https://github.com/RedHatInsights/vulnerability-ui/blob/master/src/Messages.js#L677

Reply:
Yeah, I had older version of code and when switched to master this was left over. Good catch! I'll update it.